sdk/python/tests/components/test_data/retail_product_stockout_prediction_pipeline.py

```diff
@@ -0,0 +1,64 @@
+from kfp.components import load_component_from_url
+
+automl_create_dataset_for_tables_op        = load_component_from_url('https://raw.githubusercontent.com/kubeflow/pipelines/b3179d86b239a08bf4884b50dbf3a9151da96d66/components/gcp/automl/create_dataset_for_tables/component.yaml')
```

Comment:
Is this commit SHA discoverable to our release script?

Reply:
Yes, but it won't be changes, since the image does not change.